5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Visualize the results of the KS-test
+
+# In[1]:
+
+
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(dplyr))
+
+
+# In[2]:
+
+
+output_fig_dir <- "figures"
```

Comment:
Is this whole thing written in R language but in a python file?

Reply:
good catch! it is supposed to be a `.r` file. The command `jupyter nbconvert --to python notebook.ipynb` is not compatible with r notebooks. I used this instead: `jupyter nbconvert --to=script --FilesWriter.build_directory=. *.ipynb`

Reply:
(see e5fc843707bfa7004d5c5ab9f7f66e7516c0838e)

Reply:
Oh interesting! This is very good to know!